src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,68 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
+    private static final String SUPPLY_IDENTIFICATION = "supply";
+    private static final String BUY_IDENTIFICATION = "buy";
+    private static final String CSV_DIVIDER = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        TransactionData transactionData = readAndCalculatedFromFile(fromFileName);
+
+        writeStatisticsToFile(toFileName, transactionData);
+    }
+
+    private TransactionData readAndCalculatedFromFile(String fileName) {
+        int supply = 0;
+        int buy = 0;
+        int result = 0;
```

Comment:
The `result` variable is a local variable and should not be a class field. It's better to declare it where it's needed, right before line 40.